docs/programming/review.rst

```diff
@@ -180,80 +184,48 @@ When submitting a ticket for review, the review request should contain the
 
-* The commit message should be on a single line and preferable under 100
-  characters.
-  The message should be a clearly articulated phrase, summarizing
-  changes done in the branch.
-  Further details about the changes can go in the release notes or review
-  request body.
+The message should be on a single line and preferable under 100 characters.
+The message should be a clearly articulated phrase,
+summarizing changes done in the branch.
 
-* Add the list of persons who should review the branch, using a
-  line starting with **reviewers** and followed by GitHub names or each
-  reviewer prefixed with **@**.
 
-* If required, using **depends-on** marker, add the list of reviews on which
-  this review depends and block the merge of this branch.
+Add the list of persons who should review the branch,
+using a line starting with **reviewers:** and followed by GitHub names or each
+reviewer prefixed with **@**.
+
+If required, using **depends-on** marker, add the list of reviews on which
+this review depends and block the merge of this branch.
 
 
 Merge your branch
 -----------------
 
-After the merge request and review was approved you need to merge your branch
-into master.
-
-After your review request was approved, you can send your branch to
-and conduct a test_review::
-
-    paver test_review GITHUB_PULL_REQUEST_ID
-
-This will trigger the buildbot tests for the branch and the results will be
-published in the PR.
-
-**Test failures during test review:**
+After the merge request and review was approved you should merge your branch
+using the GitHub merge button, as soon as possible.
 
-There is ticket https://trac.chevah.com/ticket/4091 where we should report any
-test failure which we suspect that is not related to our branch.
+GitHub might suggest it's own format for the merge, but we are using the
+PR title as the commit message with the PR ID appended to it.
 
-When test_review fails, you can retry just the failed builder- no need to
-trigger all the builders.
-See the "Resubmit Build:" section in the PR.
-
-**Test success during test review:**
-
-When using the GitHub merge button, use the standard merge commit format.
-
-That is, remove the (#PR_ID) from the end of the commit message.
-We only care about the Trac ID and it should be first :)
-
-Make sure to edit the commit details.
-GitHub will auto add the list of all commit messages.
+If the PR title is `[#1234] What was done in this branch` the commit message
+will be `[#1234] What was done in this branch. (#4567)`
+Where 1234 is the Trac ticket id and 4567 is the GitHub PR id::
 
 When doing manual merge using git, use squash merge and don't use the
 default commit message.
 Here is a sample command for merging branch "1234-what-was-done"::
 
     git checkout master
     git merge --no-commit --squash 1234-what-was-done
-    git commit -a -m "[#1234] What was done in this branch."
+    git commit -a -m "[#1234] What was done in this branch. (#4567)"
```

Comment:
What's the final value? Because I always tale it out, only leaving the Trac ID

Reply:
Initially, I thought it is a problem... but our automation scripts works.
For now [#1234] is the Trac ticket ID, but once we migrate to GitHub we will have the GitHub Issue ID at start and PR ID at the end.